pkg/config/application_ecs.go

```diff
@@ -34,9 +38,21 @@ func (s *ECSApplicationSpec) Validate() error {
 }
 
 type ECSDeploymentInput struct {
+	// The name of network configuration file placing in application directory.
+	VpcConfigurationFile string `json:"vpcConfigurationFile"`
```

Comment:
Is this duplicated with L49 (AwsVpcConfiguration field), or do you want to support both on purpose?

Reply:
I forgot to delete this line🙇